addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4867,21 +4867,57 @@ msgctxt "#10008"
 msgid "Play next"
 msgstr ""
 
-#empty strings from id 10009 to 10010
+#. Header label about yes/no dialog to confirm remove of part
+#: xbmc/addons/gui/GUIDialogAddonSettings.cpp
+msgctxt "#10009"
+msgid "Confirm add-on configuration remove"
+msgstr ""
+
+#. Description label about yes/no dialog to confirm remove of part, header defined by #10009
```

Comment:
Description label for yes/now dialog confirming removal of an add-on configuration